app/src/main/java/org/fossasia/openevent/general/auth/EditProfileFragment.kt

```diff
@@ -225,10 +215,11 @@ class EditProfileFragment : Fragment(), ComplexBackPressFragment {
         setTextIfNull(rootView.firstName, userFirstName)
         setTextIfNull(rootView.lastName, userLastName)
         setTextIfNull(rootView.details, userDetails)
-        setTextIfNull(rootView.phone, userPhone)
+        setTextIfNull(rootView.phone, userPhone.subSequence(userPhone.indexOf(" ") + 1, userPhone.length).toString())
```

Comment:
What?

Reply:
This code is there so that the Phone number field displays only the phone number, not the country code. The phone number and country code are separated by a space in the userPhone variable.

Reply:
This is very brittle

Reply:
Do you have any other approach to solve this problem?

Reply:
Yes, if I had time. But currently I am more focused on actual bugs and required enhancements. These issues and PRs are unneeded enhancements for now, so unfortunately I can't spend time thinking of correct solution for these when things are crashing, payments are not working and refresh token implementation is incomplete. I suggest you focus on those issues as well. If someone else wants to help you improve the logic, and it looks maintainable to me, I'll merge the PR

Reply:
Ok.